.github/workflows/csound_builds.yml

```diff
@@ -166,7 +166,7 @@ jobs:
         run: brew install bison flex asio jack
 
       - name: Configure build
-        run: cmake -B build -S . -DUSE_VCPKG=1 -DCUSTOM_CMAKE="./platform/osx/custom-osx.cmake"
+        run: cmake -B build -S . -DBUILD_TESTS=1 -DUSE_VCPKG=1 -DCUSTOM_CMAKE="./platform/osx/custom-osx.cmake"
```

Comment:
I think that most custom cmake files like`custom-osx.cmake` have the tests enabled anyway